package.json

```diff
@@ -1,11 +1,32 @@
 {
   "private": true,
+  "name": "material-motion-js",
```

Comment:
Let's keep the name out of here for now too.  This package is just to hold our scripts and common devDependencies.